app.js

```diff
@@ -38,7 +57,9 @@ export default class FlashCards extends Component {
   render() {
     return(
       <div className="container">
-        <Form value={this.state} change={this.handleChange} submit={this.handleSubmit}/>
+        <Nav />
+        <h1 id="title">React Flash Cards <i class="fab fa-react"></i></h1>
```

Comment:
React uses `className` instead of `class` for CSS classes. You should set up your linters.